examples/SPDXJSONExample-v2.0.json

```diff
@@ -2,380 +2,267 @@
+      "copyrightText" : "Copyright 2001-2011 The Apache Software Foundation",
+      "fileContributor" : [ "Apache Software Foundation" ],
+      "fileName" : "./lib-source/commons-lang3-3.1-sources.jar",
+      "fileType" : [ "ARCHIVE" ],
+      "licenseConcluded" : "Apache-2.0",
+      "licenseInfoInFile" : [ "Apache-2.0" ],
+      "noticeText" : "Apache Commons Lang\nCopyright 2001-2011 The Apache Software Foundation\n\nThis product includes software developed by\nThe Apache Software Foundation (http://www.apache.org/).\n\nThis product includes software from the Spring Framework,\nunder the Apache License 2.0 (see: StringUtils.containsWhitespace())"
+    }, {
+      "annotation" : [ {
+        "annotationDate" : "2011-01-29T18:30:22Z",
+        "annotationType" : "OTHER",
+        "annotator" : "Person: File Commenter",
+        "comment" : "File level annotation"
+      } ],
+      "checksum" : [ {
+        "algorithm" : "MD5",
+        "checksumValue" : "624c1abb3664f4b35547e7c73864ad24"
+      }, {
+        "algorithm" : "SHA1",
+        "checksumValue" : "d6a770ba38583ed4bb4525bd96e50461655d2758"
+      } ],
+      "comment" : "The concluded license was taken from the package level that the file was included in.\nThis information was found in the COPYING.txt file in the xyz directory.",
+      "copyrightText" : "Copyright 2008-2010 John Smith",
+      "fileContributor" : [ "The Regents of the University of California", "Modified by Paul Mundt lethal@linux-sh.org", "IBM Corporation" ],
+      "fileName" : "./package/foo.c",
+      "fileType" : [ "SOURCE" ],
+      "licenseComments" : "The concluded license was taken from the package level that the file was included in.",
+      "licenseConcluded" : "(LGPL-2.0-only OR LicenseRef-0)",
+      "licenseInfoInFile" : [ "GPL-2.0-only", "LicenseRef-0" ],
+      "noticeText" : "Copyright (c) 2001 Aaron Lehmann aaroni@vitelus.com\n\nPermission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the �Software�), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions: \nThe above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.\n\nTHE SOFTWARE IS PROVIDED �AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE."
+    } ],
+    "snippets" : [ {
```

Comment:
Same issue on snippets, it looks like the Snippets are missing their SPDX ID definitions.

Reply:
Added SPDX ID's in the latest commit